flutter/lib/src/sentry_flutter_options.dart

```diff
@@ -162,6 +162,15 @@ class SentryFlutterOptions extends SentryOptions {
   /// Only available on Android.
   bool enableNdkScopeSync = false;
 
+  /// The property [appStartFinish] will be automatically set in
+  /// `SchedulerBinding.addPostFrameCallback()`. Set this to false if you want
+  /// to set it yourself. It will be sent with the next transaction afterwards.
+  bool autoAppStartFinish = true;
```

Comment:
I kinda believe that `true` isn't a good default value because you definitely get a wrong measurement if a user of the library calls init after the application has been started. It would affect Sentry Mobile as well.

Maybe one can automatically detect such thing and work around it?

Reply:
I imagined two cases:

- User starts sentry right at the beginning and everything works out of the box.
- Users can opt out when configuring options and call the method optionally.

Both rely on users starting the SDK upon app start, which we for example don't do with the sentry mobile app. I'm nur sure how to detect this.

The other option would be to reverse this and make it as an opt-in option. Then we won't have incorrect data for this case.

Reply:
We could use https://api.flutter.dev/flutter/dart-ui/PlatformDispatcher/frameData.html which has the `frameNumber` as an indicator. If that's not at 0 (or whatever the start count is) you can automatically opt out and print a warning.

The drawback: It's only available after, I think, Flutter 2.5. So you would have to do something like: 
```dart
try {
  final frameNumber = (WidgetsFlutterBinding.instance!.window as dynamic).frameData.frameNumber as int;
} on NoSuchMethodError catch (_) {
  // ...
}
```

Whether that's actually worth it, is up to you.

Reply:
Ok, that would be a possibility, but might also lead to confusion to users. There could be edge cases where users would want auto, but would get cut off? Difficult to decide tbh, maybe @marandaneto can help us out here?

Reply:
I'll check how it is done on the other SDKs

Reply:
For now, we can document that people can turn it off the feature if initing the SDK too late, most probably, and later, they can add their own measurements to their own transactions anyway.
They could already do it via the `SentryMeasurement` class, so we unblock this PR, but we have to document this behavior.
TLDR: I'd rather not have a boolean called `autoAppStartFinish` but rather a flag called `autoAppStart` which enable/disable the whole feature.